vmdb/app/models/miq_database.rb

```diff
@@ -100,4 +101,11 @@ def verify_credentials(auth_type=nil, options={})
 
     MiqTask.wait_for_taskid(RegistrationSystem.verify_credentials_queue).task_results if auth_type == :registration
   end
+
+  private
+
+  def registration_server_url_is_valid
+    return if registration_type != "rhn_satellite"  # Only validating Satellite 5 URL
```

Comment:
Does this have to return `true`?  I thought returning `nil` was like a failure of the validation (I could be wrong)